projects/storefrontlib/src/layout/a11y/skip-link/component/skip-link.component.spec.ts

```diff
@@ -52,7 +54,9 @@ describe('SkipLinkComponent', () => {
 
   beforeEach(() => {
     fixture = TestBed.createComponent(SkipLinkComponent);
+    service = TestBed.get(SkipLinkService as Type<SkipLinkService>);
     skipLinkComponent = fixture.componentInstance;
+    skipLinkComponent.skipLinks$ = service.getSkipLinks(); // Don't use async pipe for test
```

Comment:
Are you able to fix the test without overwriting component property?
In the future, someone could add something more to that pipe in component, which won't cause test to fail, because we overwrite it with our assumed implementation.